static/src/javascripts/projects/common/modules/identity/consent-wizard.js

```diff
@@ -0,0 +1,105 @@
+// @flow
+
+import fastdom from 'lib/fastdom-promise';
+
+import loadEnhancers from './modules/loadEnhancers';
+import { newsletterCheckboxClassName } from './consents';
+import { wizardPageChangedEv } from './wizard';
+
+const getClickedCheckboxes = (
+    checkboxesEl: Array<HTMLLabelElement>
+): Promise<void> =>
+    fastdom.read(
+        () =>
+            checkboxesEl.filter(
+                (checkboxEl: HTMLLabelElement) =>
+                    checkboxEl.control instanceof HTMLInputElement &&
+                    checkboxEl.control.checked
+            ).length
+    );
+
+const getEmailCheckboxes = (): Promise<Array<HTMLLabelElement>> =>
+    fastdom.read(() => [
```

Comment:
ditto.

Reply:
Nope, only when reading properties that are layout-related. See the list [here](https://gist.github.com/paulirish/5d52fb081b3570c81e3a).